modules/keypad/commerce_pos_keypad.info

```diff
@@ -4,3 +4,4 @@ core = 7.x
 package = Commerce POS
 
 dependencies[] = commerce_pos
+dependencies[] = commerce_pos:commerce_pos_currency_denomination
```

Comment:
This is really cool. But according to Drupal.org, we need to also require Drupal 7.40 when we use this new format: https://www.drupal.org/docs/7/creating-custom-modules/writing-module-info-files-drupal-7x#dependencies

dependencies[] = drupal:system (>= 7.40)